tests/cover/test_numerics.py

```
@@ -23,11 +23,33 @@
 import pytest
 
 from hypothesis import given, assume
+from hypothesis.errors import InvalidArgument
 from tests.common.utils import fails
-from hypothesis.strategies import decimals, fractions
+from hypothesis.strategies import data, none, tuples, decimals, integers, \
+    fractions
 from hypothesis.internal.compat import float_to_decimal
 
 
+@given(data())
+def test_fuzz_fractions_bounds(data):
+    denom = data.draw(none() | integers(1, 100))
+    fracs = none() | fractions(max_denominator=denom) \
+        | fractions('1/99', '1/2', denom)
+    try:
+        low, high = data.draw(tuples(fracs, fracs))
+        if low is not None and high is not None and low > high:
+            low, high = high, low
+        val = data.draw(fractions(low, high, denom))
+    except InvalidArgument:
+        assume(False)  # fractions too close for given max_denominator
```

`reject()` rather than `assume(False)`